lib/govuk_publishing_components/presenters/layout_footer_helper.rb

```diff
@@ -1,35 +1,25 @@
 module GovukPublishingComponents
   module Presenters
     class LayoutFooterHelper
-      attr_reader :ga4_index_total, :ga4_index_section_count, :ga4_ogl_link_index_section, :ga4_copyright_link_index_section
+      attr_reader :ga4_index_section_count, :ga4_ogl_link_index_section, :ga4_copyright_link_index_section
 
       def initialize(footer_navigation_section, footer_meta_section)
-        @ga4_index_total = 0
-        @ga4_index_section_count = 0
+        @ga4_index_section_count = footer_navigation_section.length
         @amount_of_meta_sections = 0
 
-        footer_navigation_section.each do |section|
-          @ga4_index_section_count += 1
-          if section[:items]
-            @ga4_index_total += section[:items].length
-          end
-        end
-
         if footer_meta_section.any?
           @amount_of_meta_sections = 1
           @ga4_index_section_count += 1
-          @ga4_index_total += footer_meta_section[:items].length
         end
 
-        # For our GOVUK footer links, we need to add 2 to the index_total and index_section_count. This is due to the "Open Government License" and "Crown Copyright" links being defined separately to the rest of our footer links.
-        @ga4_index_total += 2
+        # For our GOVUK footer links, we need to add 2 to the index_section_count. This is due to the "Open Government License" and "Crown Copyright" links being defined separately to the rest of our footer links.
         @ga4_index_section_count += 2
 
         @ga4_ogl_link_index_section = footer_navigation_section.length + @amount_of_meta_sections + 1
         @ga4_copyright_link_index_section = footer_navigation_section.length + @amount_of_meta_sections + 2
```

Comment:
Quick question: this code isn't part of the PR but I had a thought whilst I was looking at this - do you think there would be any mileage in changing this to `@ga4_copyright_link_index_section = @ga4_ogl_link_index_section + 1`? If we can rely on the copyright link coming directly after the OGL link, this would update the copyright `index_section` automatically if the OGL `index_section` changed.

This is quite a nitpick though so happy to leave this as it is.

Reply:
Good shout, thanks. Have added.